module-apps/application-phonebook/windows/PhonebookNewContact.cpp

```diff
@@ -119,6 +120,16 @@ namespace gui
 
     auto PhonebookNewContact::onInput(const InputEvent &inputEvent) -> bool
     {
+        auto backToCallLogIfCameFromThere = [this]() {
+            // MOS-357: conditionally (@see ApplicationManagerCommon::handleSwitchBack) return to call log
+            return app::manager::Controller::switchBack(
+                application, std::make_unique<app::manager::SwitchBackRequest>("ApplicationCallLog"));
+        };
```

Comment:
Do we really want to mark this comment with PR number? 
How about:
```suggestion
        // Conditionally (@see ApplicationManagerCommon::handleSwitchBack) returns to the call log application
        auto backToCallLogIfCameFromThere = [this]() {
            return app::manager::Controller::switchBack(
                application, std::make_unique<app::manager::SwitchBackRequest>("ApplicationCallLog"));
        };
```

Reply:
Not a PR number, but a ticket number :) IMO this piece of code may seem weird at first glance, so I wanted to introduce a direct reference for the reader.